mypy/util.py

```diff
@@ -118,36 +118,36 @@ def try_find_python2_interpreter() -> Optional[str]:
 
 PASS_TEMPLATE = """<?xml version="1.0" encoding="utf-8"?>
 <testsuite errors="0" failures="0" name="mypy" skips="0" tests="1" time="{time:.3f}">
-  <testcase classname="mypy" file="mypy" line="1" name="mypy" time="{time:.3f}">
+  <testcase classname="mypy" file="mypy" line="1" name="mypy-py{version}-{platform}" time="{time:.3f}">
   </testcase>
 </testsuite>
 """  # type: Final
 
 FAIL_TEMPLATE = """<?xml version="1.0" encoding="utf-8"?>
 <testsuite errors="0" failures="1" name="mypy" skips="0" tests="1" time="{time:.3f}">
-  <testcase classname="mypy" file="mypy" line="1" name="mypy" time="{time:.3f}">
+  <testcase classname="mypy" file="mypy" line="1" name="mypy-py{version}-{platform}" time="{time:.3f}">
     <failure message="mypy produced messages">{text}</failure>
   </testcase>
 </testsuite>
 """  # type: Final
 
 ERROR_TEMPLATE = """<?xml version="1.0" encoding="utf-8"?>
 <testsuite errors="1" failures="0" name="mypy" skips="0" tests="1" time="{time:.3f}">
-  <testcase classname="mypy" file="mypy" line="1" name="mypy" time="{time:.3f}">
+  <testcase classname="mypy" file="mypy" line="1" name="mypy-py{version}-{platform}" time="{time:.3f}">
```

Comment:
Oh, is this field what Changes displays? I wasn't sure how Changes maps the various fields to the UI.